src/ensembl/src/content/app/browser/browser-region-editor/BrowserRegionEditor.test.tsx

```diff
@@ -2,37 +2,42 @@ import React from 'react';
 import { mount } from 'enzyme';
 import faker from 'faker';
 
-import { BrowserRegionEditor } from './BrowserRegionEditor';
+import {
+  BrowserRegionEditor,
+  BrowserRegionEditorProps
+} from './BrowserRegionEditor';
 import Input from 'src/shared/components/input/Input';
 import Select from 'src/shared/components/select/Select';
 import Tooltip from 'src/shared/components/tooltip/Tooltip';
 
-import { ChrLocation } from '../browserState';
-import genomeKaryotypes from 'tests/data/browser/karyotypes';
+import { createGenomeKaryotypes } from 'tests/fixtures/genomes';
 import { getCommaSeparatedNumber } from 'src/shared/helpers/numberFormatter';
-import { LoadingState } from 'src/shared/types/loading-state';
-import { createValidationInfo } from 'tests/fixtures/browser';
-
-const defaultProps = {
-  activeGenomeId: faker.lorem.words(),
-  chrLocation: ['13', 2315086, 32400266] as ChrLocation,
-  genomeKaryotypes,
-  regionEditorActive: false,
-  regionFieldActive: false,
-  regionValidationInfo: {},
-  regionValidationLoadingStatus: LoadingState.NOT_REQUESTED,
-  changeBrowserLocation: jest.fn(),
-  replace: jest.fn(),
-  resetRegionValidation: jest.fn(),
-  toggleRegionEditorActive: jest.fn(),
-  validateRegion: jest.fn()
-};
+import {
+  createValidationInfo,
+  createChrLocationValues
+} from 'tests/fixtures/browser';
 
 describe('<BrowserRegionEditor', () => {
   afterEach(() => {
     jest.resetAllMocks();
   });
 
+  const initialChrLocation = createChrLocationValues().tuppleValue;
```

Comment:
1) tuple, not tupple
2) perhaps `asTuple` and `asString` rather than `tupleValue` and `stringValue`?

Reply:
I was tempted to use `asString`, however that would be slightly similar to `as string` in TypeScript so wanted to stay clear of it. And as for the function name, won't `createChrLocation` just mean creating one value that is a tuple rather than having two separate versions of it?